AwfulCore/Sources/AwfulCore/Model/Awful.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Awful 6.2.xcdatamodel</string>
+	<string>Awful 6. 3.xcdatamodel</string>
```

Comment:
Weird version name, with the space before `3` :) Purely cosmetic but kinda weird.

Reply:
This should be resolved! I published a new model version with the correct naming and deleted the borked one. I've cloned this repo after pushing that change and all seems to be working OK